docs/src/Microphysics2M.md

```diff
@@ -408,53 +408,38 @@ Z_r = 3 \, \frac{6! \, A_r}{(B_r)^{7} \, (\frac{4}{3} \, \pi \, \rho_w)^{2}}.
 \end{equation}
 ```
 
-To obtain the logarithmic radar reflectivity ``L_z``, which is commonly used to refer to the radar reflectivity values, we divide both ``Z_c`` and ``Z_r`` with the equivalent return of a ``1 mm`` drop in a volume of a meter cube (``Z_0``), apply the decimal logarithm to the result, and multiply the result by ``10``. 
-
-For example, for the cloud droplets radar reflectivity we have:
+To obtain the logarithmic radar reflectivity ``L_z``, which is commonly used to refer to the radar reflectivity values, we sum ``Z_c`` and ``Z_r``, and we normalize the result with the equivalent return of a ``1 mm`` drop in a volume of a meter cube (``Z_0``). 
+Then, we apply the decimal logarithm, and multiply the result by ``10``:
 ```math
 \begin{equation}
-L_{Z_c} = 10 \log_{10}(\frac{Z_c}{Z_0}).
+L_z = 10 \log_{10}(\frac{Z_c + Z_r}{Z_0}).
 \end{equation}
 ```
 The resulting logarithmic dimensionless unit is decibel relative to ``Z``, or ``dBZ``.
 
-Lastly, the weighted average of ``Z_c`` and ``Z_r`` over their number densities (``N_{liq}`` and ``N_{rai}``) allows us to obtain the final value for ``L_z``
-```math
-\begin{equation}
-L_z = \frac{L_{Z_c} N_{liq} + L_{Z_r} N_{rai}}{N_{liq} + N_{rai}}.
-\end{equation}
-```
-
 ### Effective radius
 
-The effective radius of hydrometeors (``r_{eff}``) is the weighted average of their size distribution, and it is defined as the ratio of the third to the second moment of a droplet size distribution (``n(r)``) :
+The effective radius of hydrometeors (``r_{eff}``) is the weighted average of their size distribution, and it is defined as the ratio of the third  to the second moment of a droplet size distribution.
```

Comment:
```suggestion
The effective radius of hydrometeors (``r_{eff}``) is the area weighted radius of the population of particles.  It can be computed as the ratio of the third  to the second moment of the size distribution.
```